index.njk

```diff
@@ -2,19 +2,54 @@
 layout: 'layouts/base.njk'
 ---
 
-<h1>
-  <span class="eyebrow">Design Sprint Guide</span>
-  Design, prototype, and test ideas quickly.
-</h1>
+<main class="tbds-app-frame" "tbds-block-stack" "tbds-block-stack--bordered">
+  
+  <header class="tbds-app-frame__header  tbds-margin-block-6">
+    <section class="tbds-inline-stack tbds-inline-stack--gap-6 tbds-inline-size-80%">
+      <div class="tbds-inline-stack__item">
+        <h5 class="tbds-brand-red">Design Sprint Guide <br/>
+            by <a class="#" href="#">thoughtbot</a></h5>
+      </div>
 
-<p>A Product Design Sprint is a technique to quickly design, prototype, and test the viability of an idea, product, or feature. The design sprint consists of 5 phases (typically days), starting with design thinking and ending with a user-tested prototype.</p>
+      <div class="tbds-inline-stack__item tbds-inline-stack__item--push-start tbds-inline-stack--gap-6 tbds-font-style-normal">
```

Comment:
I don't think that `tbds-inline-stack__item` or `tbds-font-style-normal` are having any effect here, maybe best to remove them?

Reply:
I think that if remove the `<span>` inside here, and convert to using a `<ul>` instead will not only be more semantic but allow you to use `tbds-inline-stack` to space the children (`tbds-inline-stack--gap-6` has no effect without using `tbds-inline-stack` first).

e.g.

```html
<ul class="tbds-inline-stack tbds-inline-stack__item--push-start tbds-inline-stack--gap-6">
  <li><a href="#">Schedules</a></li>
  ...
</ul>
```